controllers/replicationdestination_controller.go

```diff
@@ -315,7 +315,17 @@ func (r *rsyncDestReconciler) ensureJob(l logr.Logger) (bool, error) {
 			r.job.Spec.Template.ObjectMeta.Labels[k] = v
 		}
 		backoffLimit := int32(2)
+		disabledSync := "disabled"
+		enabledSync := "enabled"
+		syncValue := *r.Instance.Spec.Rsync.DataSync
```

Comment:
@JohnStrunk any ideas here? 

When this variable is defined it causes the tests to fail.